### PR TITLE
feat: added <feature-list />

### PR DESCRIPTION
**Pull request by vbvnvmfjg**
* [Trello](https://trello.com/c/ljIsHmdB/111-create-feature-list-component)
* [Preview on Page](https://deploy-preview-110--leanwebkit.netlify.com/en/test/)
* [Preview in Guide](https://deploy-preview-110--leanwebkit.netlify.com/guide/?selectedKind=Feature%20list&selectedStory=Default&full=0&addons=1&stories=1&panelRight=1&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel)

### CODE REVIEW

File: src/client/components/feature-list/feature-list.vue [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<template>
+  <ul class="feature-list">
+    <li
+      v-for="feature in features"
+      :key="feature.id"
+      class="feature-list__item">
+      <div class="feature-item">
```
Comment (jbmoelker) [inline src/client/components/feature-list/feature-list.vue]: Still part of the same (single file) component, so still the same prefix `feature-list`.
So I'd either do:
```
.feature-list__item
  .feature-list__item-content
  .feature-list__item-title
  .feature-list__item-image
  .feature-list__item-body
```
or

```
.feature-list__item
  .feature-list__item__content
  .feature-list__item__title
  .feature-list__item__image
  .feature-list__item__body
```

File: src/client/components/feature-list/feature-list.vue [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  max-width: 100px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.feature-item__body {
+  margin-bottom: 0;
+}
+
+@media (min-width: 600px) {
```
Comment (jbmoelker) [inline src/client/components/feature-list/feature-list.vue:90]: Should we also add two in a row when possible? One in a row does create a very long page.

**Review by jbmoelker — DISMISSED** [review]
Nice work! Just a bit of BEM naming and a question :)

File: src/client/components/feature-list/feature-list.vue [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  components: { ResponsiveImage, SmartLink },
+  props: {
+    title: {
+      type: String,
+      required: false,
+      default: ''
+    },
+    features: {
+      type: Array,
+      require: true,
```
Comment (Remcoman) [inline src/client/components/feature-list/feature-list.vue]: small typo here :-)

**Review by Remcoman — CHANGES_REQUESTED** [review]

File: src/client/components/feature-list/feature-list.vue (hunk 3 above)
Comment (vbvnvmfjg) [reply src/client/components/feature-list/feature-list.vue]: I fixed the typo

**Review by Remcoman — DISMISSED** [review]

**Pull request merged**